lib/galaxy/queue_worker.py

```diff
@@ -63,6 +63,7 @@ def _get_new_toolbox(app):
     and then adding pre-existing data managers from the old toolbox to the new toolbox.
     """
     from galaxy import tools
+    from galaxy.tools.special_tools import load_lib_tools
     from galaxy.tools.toolbox.lineages.tool_shed import ToolVersionCache
```

Comment:
Damn circular imports! Sorry @mvdbeek, I hoped it was an easy fix, but @jmchilton is right, a separate PR is entirely fair.